src/modules/launcher/PowerLauncher/LauncherControl.xaml

```diff
@@ -80,7 +80,7 @@
             x:FieldModifier="public"
             Opacity="0.6"            
             Canvas.ZIndex="0"
-            Margin="24, 0, 0, 0"
+            Margin="24, 0, 14, 0"
```

Comment:
[clarification] Should we make the margin the same on both the right and the left? Was it added because otherwise the text would start too much to the right?

Reply:
Yeah it was added because otherwise the autocomplete text wouldnt get superimposed by the actual text. I think its because the left margin for the AutoCompleteBlock is 24, while for the QueryTextBox its 10, but I don't know the exact reason that was done. I would either have to change a margin on the AutoCompleteBlock or on the QueryTextBox to get them to align. I chose to do it on the autocompleteblock since there was lesser chance of it affecting anything for the Left to Right case.